src/actions/pause/commands.js

```diff
@@ -18,12 +19,11 @@ type CommandType = "stepOver" | "stepIn" | "stepOut" | "resume";
  */
 export function command(type: CommandType) {
   return async ({ dispatch, client }: ThunkArgs) => {
-    // execute debugger thread command e.g. stepIn, stepOver
-    dispatch({ type: "COMMAND", value: { type } });
-
-    await client[type]();
-
-    dispatch({ type: "CLEAR_COMMAND" });
+    return dispatch({
+      type: "COMMAND",
+      command: type,
+      [PROMISE]: client[type]()
+    });
```

Comment:
Is this neglecting to put a `status` property on this action, or is this not a COMMAND `PauseAction`?

Reply:
I see; this communicates with the promise middleware.